x-pack/plugins/osquery/cypress/e2e/all/packs_integration.cy.ts

```diff
@@ -165,8 +164,9 @@ describe.skip('ALL - Packs', { tags: ['@ess', '@serverless'] }, () => {
         cy.contains('Run a set of queries in a pack.').click();
         cy.getBySel(LIVE_QUERY_EDITOR).should('not.exist');
         cy.getBySel('globalLoadingIndicator').should('not.exist');
-        cy.getBySel('select-live-pack').click().type('osquery-monitoring{downArrow}{enter}');
         selectAllAgents();
+        cy.getBySel('select-live-pack').click();
+        cy.getBySel('select-live-pack').type('osquery-monitoring{downArrow}{enter}');
```

Comment:
Q: With this change is the `.click` really necessary? I would guess that just finding the input and "type"ing should work. Else  I would imagine you need to chain the `type` to `click` with a `then`. 🤔

Reply:
Thanks @ashokaditya 👍 
1. I think testing flakiness against serverless is what we were looking for, this particular test didn't fail on ESS. 
2. I'll try removing the `.click()`, but from my experience, clicking the element gives us more time for the input to load, therefore reduces the flakiness. I would prefer not to change it tbh. I think that `.then()` is not needed, Cypress automatically wait for the first command to finish after going to the other one.